submitter/sqs.py

```diff
@@ -48,13 +49,23 @@ def process(msgs):
             submission = Submission.from_message(message)
             if not submission.result_message:
                 submission.submit(client)
-            write_message_to_queue(
+            response = write_message_to_queue(
                 submission.result_attributes,
                 submission.result_message,
                 submission.result_queue,
             )
-        # TODO: probs better to confirm the write to the output was good
-        # before cleanup but for now yolo it
+            if not verify_sent_message(submission.result_message, response):
+                raise errors.SQSMessageSendError(
```

Comment:
Will this halt the program or will the rest of the messages in the queue continue to be processed?

Reply:
It will halt the program. My reasoning being that we don't want it to keep processing and possibly re-publishing the item from the original submit message that's still in the submit queue. Instead we want to stop, investigate, deal with any issues, and then restart. Thoughts?

Reply:
Halting in this case is the correct behavior. Continuing to process messages when we had a failure to write to the output queue for any reason is def concerning and we should stop and debug. 👍🏻